.github/CODEOWNERS

```diff
@@ -133,19 +152,31 @@ CHANGELOG*
 /x-pack/filebeat/module/threatintel @elastic/security-external-integrations
 /x-pack/filebeat/module/tomcat @elastic/security-external-integrations
 /x-pack/filebeat/module/zeek @elastic/security-external-integrations
-/x-pack/filebeat/module/zookeeper @elastic/integrations
+/x-pack/filebeat/module/zookeeper @elastic/obs-infraobs-integrations
 /x-pack/filebeat/module/zoom @elastic/security-external-integrations
 /x-pack/filebeat/module/zscaler @elastic/security-external-integrations
 /x-pack/filebeat/processors/decode_cef/ @elastic/security-external-integrations
 /x-pack/heartbeat/ @elastic/uptime
 /x-pack/metricbeat/ @elastic/elastic-agent-data-plane
 /x-pack/metricbeat/docs/ # Listed without an owner to avoid maintaining doc ownership for each input and module.
 /x-pack/metricbeat/module/ @elastic/integrations
+/x-pack/metricbeat/module/activemq @elastic/obs-infraobs-integrations
+/x-pack/metricbeat/module/airflow @elastic/obs-infraobs-integrations
+/x-pack/metricbeat/module/cloudfoundry @elastic/obs-infraobs-integrations
```

Comment:
how about `/x-pack/metricbeat/module/aws`, `/x-pack/metricbeat/module/awsfargate`, `/x-pack/metricbeat/module/azure` and `/x-pack/metricbeat/module/gcp`?

Reply:
Not covering cloud packages in this PR currently. @tommyers-elastic, Can we create separate PR for for the cloud packages?

Reply:
We will file separate PRs for the cloud filebeat/metricbeat changes.